doc/sphinx/administration_portal/client/vpbx/routing_endpoints/ivrs.rst

```diff
@@ -81,8 +107,26 @@ an error and will trigger the **Error process**):
 
 .. rubric:: Entries are regular expressions
 
+*****************
+Entry information
+*****************
 You can specify IVR entries as Regular Expressions. If entry is just
 a numeric value, it will be handled as a sequence of digits, otherwise it
-will be handled a regular expression. This can be handy if you have the
+will be handled as a regular expression. This can be handy if you have the
 same behaviour for a group of dialed numbers.
 
+    Entry
+        Sequence of digits or regular expression.
```

Comment:
It's an actual regular expression only value, so I would remove sequence here to avoid extra confusion.

What is more, if you configure `1`  and `11` as two different entries, the second one will never match (because 1 regexp matches if someone enters 11)